articles/article-28-machine-language-vs-programming-language.md

```diff
@@ -0,0 +1,19 @@
+<!-- BEGIN TITLE -->
+# Machine Language vs. Programming Language
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+A programming language is a language that programmers use to develop applications. It is also called high-level language as they are more human readable.  Examples of such programming language include -but not limited to- C++, Java, and Fortran.
+
+On the other hand, machine language is a low-level language, because the computer can understand and process directly without any transformations. It is only one language which is a stream of 0's and 1's.
```

Comment:
One gets the impression, that machine language is the only type of low level language. But there are others, such as for example assembler or even, to some degree c.